src/Tasks/ManifestUtil/SecurityUtil.cs

```diff
@@ -205,7 +205,9 @@ private static XmlElement GetXmlElement(string targetZone, FrameworkName fn)
                         {
                             try
                             {
+#pragma warning disable CA2000 // Dispose objects before losing scope is suppressed because it is managed by FileStream above.
                                 var sr = new StreamReader(fs);
```

Comment:
Here I think it would be better to fix it by constructing the `StreamReader` with `leaveOpen: true`. The way it is now, if `sr` gets collected before we leave this method it will close `fs` prematurely.